dataio/dataio.py

```
@@ -135,13 +159,17 @@ def audio_pipeline(Path, Label, SampleRate, AudioChannel):
     return train_data, valid_data, train_loader_kwargs, valid_loader_kwargs
 
 def dataio_prepare_test(hparams):
-    """prepare test dataset
+    """Prepare test dataset.
     """
     data_folder = hparams["data_folder"] 
     test_data = sb.dataio.dataset.DynamicItemDataset.from_csv(
         csv_path=hparams["test_csv"],
         replacements={"ROOT": data_folder},
     )
+    # We only filter out very short audio in testing
+    test_data = filter_by_duration(
```

For this one, I suggest not filtering out test set utterances. 
Everything test file should receive a score, whether it is long or short; otherwise, users may unaware that some of the test utterances are not scored. The results may not be compatible if users use different parameter.
 
A too-short utterance may raise error (not long or produce a single frame), but we can just let it raise an error, or put an warning here. The thing is to let the user know, rather than drop it silently.

---

I will close the current pull request. It is better to show a warning at dataio.